cart.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://artlyui.netlify.app/stylesheets/main.css"/>
+        <link rel="stylesheet" href="./stylesheets/styles.css"/>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" crossorigin="anonymous" referrerpolicy="no-referrer"/>
+        <title>Artly Store</title>
+    </head>
+    <body>
+        <div class="cart-wrapper">
+        <header class="header">
+            <div class="logo-container">
+                <a href="./index.html">
+                <h1>Artly<span class="logo-text">Store</span></h1></a>
+            </div>
+            <div class="search-wrapper">
+            <form>
+              <input
+                type="text"
+                name="search-bar"
+                id="search-bar"
+                placeholder="Search"
+              />
+            </form>
+            </div>
+            <button class="btn btn-primary">Login</button>
+            <div class="header-nav">
+                <nav>
+                    <ul>
+                        <li>
+                            <div class="badge-container">
+                                <span class="badge badge-sm badge-text">0</span>
+                                <img class="img-xs" src="./assets/heart.svg" alt="shopping-cart"/>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="badge-container">
+                                <span class="badge badge-sm badge-text">0</span>
+                                <img class="img-xs" src="./assets/cart.svg" alt="shopping-cart"/>
+                            </div>
+                        </li>
+                    </ul>
+                </nav>
+                <a href="#" class="user">
+                  <img class="img-sm-2" src="./assets/user.svg" alt="user"/>
+                </a>
+            </div>
+        </header>
+        <main>
+          <div class="main-section">
```

Comment:
Just a suggestion you can use section tag here

Reply:
Got it. I have addressed this.